packages/components/inputs/selectable-search-input/src/selectable-select.tsx

```diff
@@ -50,23 +51,6 @@ const SelectableSelect = (props: TSelectableSelect) => {
     dropdownHasFocus: props.dropdownHasFocus,
   }) as ReactSelectProps['styles'];
 
-  const { onChange, name, id, textInputRef } = props;
```

Comment:
Do we really need to mode this code to the parent?

Perhaps we can just keep this here and, in the parent, listen to the `onChange` event, update the selected dropdown state and bubble up the event.

Reply:
Thank you for the observation, I actually noticed all other methods ( `handleContainerBlur`, `handleDropdownBlur`, `handleDropdownFocus`) are calculated in the `selectable-search-input` and then passed to the `selectable-select` as props, except the `handleDropdownChange`. It felt a bit inconsistent to me. That's why I moved it there. 
I didn't see a practical reason to leave it there. Either options are still fine by me. What do you think?

Reply:
Fair point 👍